### PR TITLE
fix: Generate service account based purely on namespace

### PR DESCRIPTION
**Pull request by jsbroks**

### CODE REVIEW

**Review by vanpelt — APPROVED** [review]
😎

File: examples/byob/main.tf [hunk 1]
```diff
@@ -34,7 +34,7 @@ module "resources" {
 resource "google_storage_bucket_iam_member" "admin" {
   bucket = module.resources.bucket_name
   member = "serviceAccount:${var.service_account_email}"
-  role   = "roles/storage.admin"
+  role   = "roles/storage.objectAdmin"
```
Comment (elainaRenee) [inline examples/byob/main.tf:37]: I thought the service account needs `storage.admin` permissions to create the bucket?

Comment (jsbroks) [reply examples/byob/main.tf:37]: The account running this tf needs storage.admin to create the account. We need storage.objectAdmin to access the bucket.

**Comment by jsbroks** [comment]
image
This wont work

**Pull request closed without merging**